mw-webapp/src/component/userCard/UserCard.module.scss

```diff
@@ -90,5 +90,8 @@ $userCardAdditionalHeight: 90px;
 
 .cardLink {
   font-size: $fontSizeBig;
-  text-decoration: none;
```

Comment:
If you delete this how will be look user card with hover? All text inside will be underlined?

Reply:
It won't be underlined, the 95th row below prevents from this behavior